src/index.html

```diff
@@ -17,6 +17,93 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="logo_link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="#apple"
+              class="is-active nav_link"
```

Comment:
The class `nav_link` should be `nav__link` to follow the BEM naming convention. Also, the class `is-active` should be `nav__link--active` if it's a modifier for `nav__link`.

Reply:
👍